modules/ROOT/pages/apikit-4-soap-project-task.adoc

```diff
@@ -1,27 +1,33 @@
-= To Create an APIkit for SOAP Project
+= Create an APIkit for SOAP Project
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You create a new project, specify the use of APIkit components, and refer to a WSDL file that you download.
+This process walks you through the creation of a create a new Mule project based on a SOAP API specification.
 
-. Under the File menu, select New > Mule Project.
+Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[Prerequisites for Using APIkit for SOAP]
+
+. Under the File menu, select *New* > *Mule Project*.
 +
-The New Mule Project wizard appears.
-. In Project Name, enter a name for the project, for example *myproject*. Select or accept the default EE Runtime, for example Mule Server 4.1.0 EE.
-. In API Implementation, check Specify API Definition File Location or URL, and browse to the WSDL file you downloaded earlier.
+image::create-new-mule-project-soap.png[]
+<1> In *Project Name*, enter a name for the project. +
+For example _soap-implementation_.
+<1> Select the Mule runtime engine version. +
+For example `Mule Server 4.2.2 EE`.
+<1> Under *API Specification*, select the  *From Design Center or Local Files* tab.
+<1> Select the expand icon: and select *Browse files...*.
```

Comment:
The colon should not be there. I suggest: "Select the expand icon and then select..."

Reply:
Thanks! I missed the image of the icon there!